Dockerfile

```diff
@@ -32,18 +32,21 @@ ENV GO111MODULE=on
 RUN go build -mod=readonly "-ldflags=-s -w" ./...
 
 ARG VERSION
+ARG BACKUP_AGENT_IMAGE
 ARG RESTORE_AGENT_IMAGE
 
 # Compile all the binaries
 RUN go build \
     -mod=readonly \
-    -ldflags="-X=github.com/improbable-eng/etcd-cluster-operator/version.Version=${VERSION} -X=main.defaultRestoreAgentImage=${RESTORE_AGENT_IMAGE}" \
+    -ldflags="-X=github.com/improbable-eng/etcd-cluster-operator/version.Version=${VERSION}\
+              -X=main.defaultBackupAgentImage=${BACKUP_AGENT_IMAGE}\
+              -X=main.defaultRestoreAgentImage=${RESTORE_AGENT_IMAGE}" \
```

Comment:
The backup-agent image name is stamped into the manager binary, and is then used as the default value of the `--backup-agent-image` flag.

This saves having to modify this flag with kustomize, which would be a pain.